src/cache/diagnostic_edmf_precomputed_quantities.jl

```diff
@@ -310,6 +314,19 @@ function set_diagnostic_edmf_precomputed_quantities!(Y, p, t)
                 dt,
             )
 
+            @. nh_pressureʲ_prev_level = ᶠupdraft_nh_pressure(
+                params,
+                p.atmos.edmfx_nh_pressure,
+                local_geometry_prev_halflevel,
+                -∇Φ³_prev_level * (ρʲ_prev_level - ρ_prev_level) / ρʲ_prev_level,
+                u³ʲ_prev_halflevel,
+                u³⁰_prev_halflevel,
+                FT(8000),
```

Comment:
I will add a todo that it is better to use updraft top here.

Reply:
Can you at least make it a parameter for now, rather than a fixed number? And setting it to the scale height (RT/g with a reference T) would make sense.

Reply:
I can set it to R_d * T_surf_ref / g then. All these three constants are in CLIMAParameters. Does that sound good to you?

Reply:
Yes